src/UI/HTMLTooltip.js

```diff
@@ -27,14 +27,14 @@ export const defaultOptions = {
     wrapperClass: '',
     tooltipPositionClass: (top, left) => `
         .tooltip {
-            transform: translate(${left}px, ${top}px) !important;
+            transform: translate(${Math.floor(left)}px, ${Math.floor(top)}px) !important;
         }
     `,
     caretPositionClass: (top, left, isAboveInput) => `
         .tooltip--email__caret {
             ${isAboveInput
-        ? `transform: translate(${left}px, ${top}px) rotate(180deg); transform-origin: 16px !important;`
-        : `transform: translate(${left}px, ${top}px) !important;`
+        ? `transform: translate(${Math.floor(left)}px, ${Math.floor(top)}px) rotate(180deg); transform-origin: 18px !important;`
+        : `transform: translate(${Math.floor(left)}px, ${Math.floor(top)}px) !important;`
```

Comment:
All this stuff and other CSS changes are of course unrelated. Just squeezing in a 2px reduction in icon size requested by the design folks. The `floor` is just to avoid weird artefacts in edge cases.